file-formats/uipg/type/zif_aff_uipg_v1.intf.abap

```diff
@@ -132,13 +160,21 @@ INTERFACE zif_aff_uipg_v1
       id              TYPE c LENGTH 35,
 
       "! <p class="shorttext">Type</p>
-      "! Type
+      "! type
```

Comment:
I still would start with a capital letter for the first word.

```suggestion
      "! Type
```

Reply:
done